script/gather_explorer_stake_accounts.rb

```diff
@@ -21,6 +21,8 @@ def new_epoch_detected?(network, last_epoch)
     demo: false,
     logger: logger
   ).call
+
+  CreateVoteAccountStakeHistoryService.new(network: network, epoch: last_epoch).call
```

Comment:
If gathering stake accounts fails because of timeout, this code is executed anyway. As a result, when script runs for the second time, it throws a RecordNotUnique error:
```
Mysql2::Error: Duplicate entry '577-mainnet' for key 'explorer_stake_account_history_stats.index_explorer_stake_account_history_stats_on_epoch_and_network' (ActiveRecord::RecordNotUnique)
```